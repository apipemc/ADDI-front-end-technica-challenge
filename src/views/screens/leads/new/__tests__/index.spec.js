import React from 'react';
import { shallow } from 'enzyme';

import ScreenNew from '../index';

describe('ScreenNew', () => {
  const props = {
    history: { push: jest.fn() },
  };

  it('should render a view', () => {
    const wrapper = shallow(<ScreenNew {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
