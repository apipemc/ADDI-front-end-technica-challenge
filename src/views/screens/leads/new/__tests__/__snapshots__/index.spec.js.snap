// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScreenNew should render a view 1`] = `ShallowWrapper {}`;
